library/common/extensions/filters/http/platform_bridge/filter.cc

```diff
@@ -559,7 +559,7 @@ void PlatformBridgeFilter::ResponseFilterBase::resumeIteration() {
 // this shortcut works for now.
 Buffer::Instance* PlatformBridgeFilter::RequestFilterBase::buffer() {
   Buffer::Instance* internal_buffer = nullptr;
-  if (parent_.decoder_callbacks_->decodingBuffer()) {
+  if (iteration_state_ == IterationState::Stopped && parent_.decoder_callbacks_->decodingBuffer()) {
```

Comment:
do you mind leaving a comment that refreshes the reader on why a iteration state needs to be stopped?